### ISSUE TITLE
Post tweet text without image/media.

### ISSUE THREAD

**Issue by onchodila** [FixTweet/FixTweet#133]
I wish fxtwitter had a setting for only posting the text of a tweet and hide the image and maybe video, It should also work with translation of the text.

Sometimes you want to post a tweet because of the text or translation of the text but then discord AI decides that the image is explicit and just hides the embed entirely, image text and all but the link still works. There are probably other reasons why you only want to post the text, maybe the image distracting from the text or irrelevant?

Basically the opposite of `d.fxtwitter.com`.

**Comment by dangeredwolf** [comment]
Implemented `t.fxtwitter.com`
